spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -44,6 +44,7 @@
     when_I_view_the_step_by_step_page
     then_I_can_see_a_summary_section
     and_I_can_edit_the_summary_section
+    and_I_can_see_a_sidebar_settings_section_with_link "Edit"
```

Comment:
I think there should be a step after this that checks that when the "Edit" link is clicked, the user is taken to the "Choose Navigation" page, so we can be sure that we're linking to the right place.

Reply:
👍 I'll add that as an expectation inside `and_I_can_see_a_sidebar_settings_section_with_link` (don't want to bloat the scenario too much)

Reply:
Hmmm I'm already asserting that the link has the right URL:

`expect(page).to have_link(link_text, :href => step_by_step_page_navigation_rules_path(@step_by_step_page))`

I want to test that the link text says "Edit" or "View" (unlike the `and_I_can_edit_the_summary_section` above, which should only have "Edit" or nothing at all). So we'd get into a situation where when it's editable, we write:

```
and_I_can_see_a_sidebar_settings_section
and_it_should_have_a_link "Edit"
and_it_should_take_me_to_the_sidebar_settings_page
```

vs when it's not editable

```
and_I_can_see_a_sidebar_settings_section
and_it_should_have_a_link "View"
and_it_should_take_me_to_the_sidebar_settings_page
but_I_should_not_be_able_to_edit_it
# which duplicates some of our other tests
```

I think what we have already is less verbose and easier to understand.